config/utils_test.go

```diff
@@ -28,7 +28,7 @@ func TestParseConfig(t *testing.T) {
 
 func TestSetConfigDefaults(t *testing.T) {
 	cfg := &ExtensionConfig{
-		Name: "test",
+		Name: "haproxy",
```

Comment:
Can we add more tests here to check the different config default setting on the backend?